drivers/power/ltp8800/ltp8800.c

```diff
@@ -0,0 +1,1024 @@
+/*******************************************************************************
+*   @file   ltp8800.c
+*   @brief  Source code of the LTP8800 Driver
+*   @authors Cedric Justine Encarnacion (cedricjustine.encarnacion@analog.com)
+********************************************************************************
+* Copyright 2024(c) Analog Devices, Inc.
+*
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
```

Comment:
please modify all the licenses to the new BSD 3-clause license you can find in any file on the main branch